boltdesignsystem/bolt
# DS-808 Profile edit form mockup

## Jira

https://pegadigitalit.atlassian.net/browse/DS-808

## Summary

A mockup page was created in the User folder -> `/pattern-lab/?p=pages-user-profile-edit`

## Details

A Profile Edit Form was created similar to the design attached in the  [Jira task](https://pegadigitalit.atlassian.net/browse/SN-243). The card's design is omitted, instead, a simple page with form inputs is created.

## How to test

Pull the branch. Make sure that the form is created in a proper way and all necessary inputs are included.
